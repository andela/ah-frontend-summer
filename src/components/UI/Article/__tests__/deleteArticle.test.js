import React from 'react';
import { shallow } from 'enzyme';

import DeleteArticleButtons from '../DeleteArticle';

describe('<DeleteArticleButtons />', () => {
    it('DeleteArticleButtons view will render without crashing', () => {
        let wrapper = shallow(<DeleteArticleButtons />);
        expect(wrapper).toMatchSnapshot();
    });
});
