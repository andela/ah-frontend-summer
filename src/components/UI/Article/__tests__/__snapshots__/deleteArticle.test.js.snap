// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteArticleButtons /> DeleteArticleButtons view will render without crashing 1`] = `
<Container
  textAlign="center"
>
  <div>
    <p>
      Are you sure you want to delete this article?
    </p>
    <Button
      as="button"
      color="red"
    >
      Yes
    </Button>
    <Button
      as="button"
      color="blue"
    >
      No
    </Button>
  </div>
</Container>
`;
